healthcareai/tests/test_deploy_supervised_model_class.py

```diff
@@ -120,3 +121,22 @@ def runTest(self):
 
     def tearDown(self):
         del self.o
+
+
+class TestValidateDestinationTableConnection(unittest.TestCase):
+    def test_raises_error_on_table_not_existing(self):
+        try:
+            result = validate_destination_table_connection('localhost', 'foo', 'bar', 'baz')
+            print(result)
+            self.fail()
```

Comment:
Interesting and logical. Would you mind adding a comment above self.fail() so the logic doesn't have to be sorted out whenever someone finds it?